hackathon_site/hardware/api_urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path, include
+from hardware import views
+
+
+app_name = "hardware"
+
+urlpatterns = [
+    path("hardware/", views.HardwareListView.as_view()),
```

Comment:
you should add a `name` attribute to each of these paths:

```python
path("hardware/", views.HardwareListView.as_view(), name="hardware-list",),
path("categories/", views.CategoryListView.as_view(), name="category-list"),
```

I like to name resource endpoints with plurals, but keep the names singular like that. Read more about why you want to use `name` here: https://docs.djangoproject.com/en/3.0/topics/http/urls/#naming-url-patterns